### ISSUE TITLE
I have primary domain with other vendor, can I get SSL for subdomain in Azure key vault. Is there any possible way ? 

### ISSUE THREAD

**Issue by Shanmuganath** [rcl-letsencrypt-auto-ssl/Lets-Encrypt-Auto#22]
Please let me know can I get wild card SSL in Azure key vault.

my username : shan@meeraskillangels.onmicrosoft.com

**Comment by rcladmin** [comment]
You can create a wild card domain using the DNS + Key Vault method. For example, *.contoso.com, can be created.

**Comment by Shanmuganath** [comment]
I tried creating *.demo.example.com I get this following error.  (I have renamed the domain)

Error:Failed to create certificate with target Azure DNS + Key Vault for host : *.demo.example.com with Azure KeyVault. Could not validate domain for *.demo.example.com with challenge dns-01, status reported : Invalid. {"Type":"urn:ietf:params:acme:error:dns","Detail":"DNS problem: NXDOMAIN looking up TXT for _acme-challenge.demo.example.com - check that a DNS record exists for this domain","Identifier":null,"Subproblems":null,"Status":400}

Is there any changes I need to do in primary domain DNS entry ?
